apps/livreGUI/clipPlanesController/ClipPlanesController.cpp

```diff
@@ -0,0 +1,207 @@
+/* Copyright (c) 2015-2016, EPFL/Blue Brain Project
+ *                          ahmet.bilgili@epfl.ch
+ *
+ * This file is part of Livre <https://github.com/BlueBrain/Livre>
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License version 3.0 as published
+ * by the Free Software Foundation.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this library; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+#include "ClipPlanesController.h"
+
+#include <livreGUI/Controller.h>
+#include <livreGUI/ui_ClipPlanesController.h>
+
+#include <livre/core/render/ClipPlanes.h>
+
+namespace livre
+{
+
+struct ClipPlanesController::Impl
+{
+    Impl( ClipPlanesController* parent, Controller& controller )
+        : _parent( parent )
+        , _controller( controller )
+    {
+        _ui.setupUi( parent );
+
+        QWidget *slidersWidget = _ui.clipPlanesWidget;
+        slidersWidget->setEnabled( false);
+
+        parent->connect( parent,
+            static_cast<void (ClipPlanesController::*)()>(&ClipPlanesController::clipPlanesReceived),
```

Comment:
I did similar to renderParametersController. It did work so I did not change it, but I can try removing